deepfence_worker/tasks/reports/xlsx.go

```diff
@@ -150,7 +151,7 @@ func vulnerabilityXLSX(ctx context.Context, params utils.ReportParams) (string,
 				log.Error().Err(err).Msg("error generating cell name")
 			}
 			value := []interface{}{
-				nodeScanData.ScanInfo.UpdatedAt,
+				time.UnixMilli(nodeScanData.ScanInfo.UpdatedAt).String(),
```

Comment:
Minor optimization... 
We can take the time value in a variable as follows:
`updatedAt := time.UnixMilli(nodeScanData.ScanInfo.UpdatedAt).String()`
And use that variable in the calls.